botx/clients/methods/errors/stickers/sticker_pack_not_found.py

```diff
@@ -0,0 +1,49 @@
+"""Definition for "sticker pack not found" error."""
+from typing import NoReturn
+from uuid import UUID
+
+from pydantic import BaseModel
+
+from botx.clients.methods.base import APIErrorResponse, BotXMethod
+from botx.clients.types.http import HTTPResponse
+from botx.exceptions import BotXAPIError
+
+
+class StickerPackNotFoundError(BotXAPIError):
+    """Error for raising when sticker pack not found."""
+
+    message_template = "sticker pack {pack_id} not found"
+
+    #: ID of sticker pack that was requested.
```

Comment:
From `mixins/requests/stickers.py`: `id of sticker pack`. Make docstrings the same.